scuba/__main__.py

```diff
@@ -56,11 +62,11 @@ def parse_scuba_args(argv):
             help='Environment variables to pass to docker')
     ap.add_argument('--entrypoint',
             help='Override the default ENTRYPOINT of the image')
-    ap.add_argument('--list-aliases', action='store_true',
-            help=argparse.SUPPRESS)
-    ap.add_argument('--list-available-options', action=ListOptsAction,
-            help=argparse.SUPPRESS)
-    ap.add_argument('--image', help='Override Docker image')
+
+    def _list_images_completer(**_):
```

Comment:
Nit-pick: Can you move these two `_completer` functions up a little bit so we can keep all of the `ap.add_argument()` calls together? I get you were trying to keep them close to the argument that uses them, but I think it's more likely we'd want to look at all of the arguments.

Reply:
Addressed by TimothySprague/scuba@ea1be7f6fb2a25a586329f831373ff28e7a0a2db